openassessment/xblock/static/css/table_builder.css

```diff
@@ -207,7 +207,17 @@
 }
 
 .oa-table-container .oa-table-default th {
-    background-color: #00AFAA;
+    background-color: #0a267a;
```

Comment:
Is it possible to have variables for these color variants?
Rest looks good to me!

Reply:
As in the whole css file we are using this approach so that's why we will continue here with the same approach and not introducing the variables in it for now.